doc/source/locale/ja_JP/LC_MESSAGES/index.po

```diff
@@ -215,3 +139,176 @@ msgstr ""
 #~ "<https://github.com/xorbitsai/inference/pull/402>`_"
 #~ msgstr ""
 
+#~ msgid "Xorbits Inference: Model Serving Made Easy🤖"
```

Comment:
For this comment messages, I think we can remove them.

Reply:
Let's do it later.

Reply:
Cleaned